runtime/doc/news.txt

```diff
@@ -268,6 +268,7 @@ LSP
 • |Client:stop()| now accepts a numerical `force` argument to be interpreted as the time to wait
   before forcing the shutdown.
 • Add cmp field to opts of |vim.lsp.completion.enable()| for custom completion ordering.
+• Add preselect field to opts of |vim.lsp.completion.enable()| for LSP completion preselect.
```

Comment:
nit:
```suggestion
• Add preselect field to opts of |vim.lsp.completion.enable()| for completion preselect.
```
When we reference other LSP concepts in this list we don't include "LSP" since it's already implicit as an item of the LSP section.